tests/components/withings/test_binary_sensor.py

```diff
@@ -17,18 +19,18 @@ async def test_binary_sensor(
     hass: HomeAssistant,
     withings: AsyncMock,
     disable_webhook_delay,
-    config_entry: MockConfigEntry,
+    webhook_config_entry: MockConfigEntry,
     hass_client_no_auth: ClientSessionGenerator,
 ) -> None:
     """Test binary sensor."""
     await enable_webhooks(hass)
-    await setup_integration(hass, config_entry)
+    await setup_integration(hass, webhook_config_entry)
 
     client = await hass_client_no_auth()
 
     entity_id = "binary_sensor.henk_in_bed"
 
-    assert hass.states.get(entity_id).state == STATE_UNAVAILABLE
+    assert hass.states.get(entity_id).state == STATE_UNKNOWN
```

Comment:
Why is this changing?

Reply:
This binary sensor is a flaky one. It only works via webhooks, so if you don't use webhooks this doesn't work. I want to replace it with an event entity afterwards

Reply:
So what is this exactly testing given it was unavailable and now it changes?

Reply:
Previously it was unavailable since there was no data for this in the object from the start, but now it has data, hence it's now unknown